build/targetsTest.props

```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
-    <TestTargets Condition="'$(LocalBuild)' != 'True'">net462;net472;netcoreapp2.1;net6.0;net8.0</TestTargets>
-    <TestTargets Condition="'$(LocalBuild)' == 'True'">netcoreapp2.1;net8.0</TestTargets>
+    <TestTargets Condition="'$(LocalBuild)' != 'True'">net462;net472;net6.0;net8.0</TestTargets>
+    <TestTargets Condition="'$(LocalBuild)' == 'True'">net8.0</TestTargets>
     <TestTargets Condition="'$(TargetNet9)' == 'True'">$(TestTargets);net9.0</TestTargets>
-    <TestOnlyCoreTargets>netcoreapp2.1</TestOnlyCoreTargets>
```

Comment:
got an issue in VS that appears new that netcoreapp2.1 is no longer supported, and netcoreapp3.1 is out of support as wel.